src/components/ExpandablePanel.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="panel">
+    <div class="header" @click="toggleOpen">
+      {{name}}
+    </div>
+    <slot v-if="open"/>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class ExpandablePanel extends Vue {
+  @Prop() name!: string;
```

Comment:
Just for consistency with other components and to enforce a good code style, I think it could be good to change this to:
```typescript
@Prop({ required: true }) readonly name!: string
```